src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -2316,46 +2321,55 @@ namespace SystemAvailabilityManager {
 		Real64 NumHoursBeforeOccupancy; // Variable to store the number of hours before occupancy in optimum start period
 		bool exitLoop; // exit loop on found data
 
+		auto & OptStartMgr( OptStartSysAvailMgrData( SysAvailNum ) );
```

Comment:
While under the hood, this is a pointer, we typically refer to this as a [reference](http://www.tutorialspoint.com/cplusplus/cpp_references.htm).  The word pointer raised my eyebrows, but when I saw it was a reference, I lowered them back down.  The change to a reference is a nice clean up of the code.  Possibly for performance, but even if not, it helps readability.